package.json

```diff
@@ -35,10 +35,10 @@
     "@angular/platform-browser": "10.2.3",
     "@angular/platform-browser-dynamic": "10.2.3",
     "@angular/router": "10.2.3",
-    "@clr/angular": "4.0.9",
-    "@clr/core": "4.0.9",
-    "@clr/icons": "4.0.9",
-    "@clr/ui": "4.0.9",
+    "@clr/angular": "4.0.10",
```

Comment:
Do we plan on upgrading vcd_ui? Do we need to?

Reply:
I was trying to see if the Clarity bug with the multiselect combobox was fixed (it wasn't). I'm going to keep the version bumped, but no reason to bump it in VCD_UI. Although, I am going to update package.json to say we require Clarity v4